Packs/Malwarebytes/README.md

```diff
@@ -0,0 +1,11 @@
+Malwarebytes Cloud integrates with Demisto for interactions with Incident Response, Endpoint Protection, and Endpoint Protection and Response products. Malwarebytes makes endpoints resilient through adaptive cyber protection, active threat response and orchestrated endpoint control that quickly restores workforce productivity when a compromise occurs. Our adaptive cyber protection predicts and detects attacks with multi-layer detection across the kill chain. We enable active threat response where machine learning is actionable and automated allowing for full recovery when a compromise occurs. We orchestrate command and control across siloed IT and Security organizations simplifying security management and making response effective. Malwarebytes makes endpoints resilient, so that you can protect, remediate, and regain control of your digital business.  

```

Comment:
```suggestion
Malwarebytes Cloud integrates with Cortex XSOAR for interactions with Incident Response, Endpoint Protection, and Endpoint Protection and Response products. Malwarebytes makes endpoints resilient through adaptive cyber protection, active threat response and orchestrated endpoint control that quickly restores workforce productivity when a compromise occurs. Our adaptive cyber protection predicts and detects attacks with multi-layer detection across the kill chain. We enable active threat response where machine learning is actionable and automated allowing for full recovery when a compromise occurs. We orchestrate command and control across siloed IT and Security organizations simplifying security management and making response effective. Malwarebytes makes endpoints resilient, so that you can protect, remediate, and regain control of your digital business.  

```